sre-recipes/recipes/ratings_freshness_recipe/ratings_freshness_recipe.py

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf-8 -*-
+"""
+This module contains the implementation of rating_freshness recipe
+"""
+
+import logging
+import subprocess
+from recipe import Recipe
+
+
+class RatingsFreshnessRecipe(Recipe):
+    """
+    This class implements 'rating_freshness' which purposefully
+    stops recollect API calls to rating service.
+    """
+
+    name = "rating_freshness"
```

Comment:
the other active recipe is called `recipe0`. I think that naming scheme was chosen to not spoil what the cause of the error was?

I'm open to changing the naming scheme, but we should keep it consistent

Reply:
it is not easy to find a balance about the abstraction level. you [said it your self](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/632#issuecomment-770114127). i am OK to abstract the name of the recipe. @simonz130 I forgot to ask do you have some guidelines about the recipe design or wording?

Reply:
if we want to abstract the recipes and avoid repetition, the naming should be randomizes. looking on the provided options: define a custom name and ability to deactivate specific recipe, it looks like the intention is to allow to a user some way to know what to expect. at the end, anyone can have a look into the code and learn what was broken.
would it be ok to rename it to "recipe_data_freshness"?

Reply:
The original intention was to use numbers and let users find the cause themselves using the monitoring dashbaords, but we may want to rethink that at some point. My vote for this PR would be to keep at as `recipe1` to match the existing name format, and then we can re-visit the naming choice later as we roll out more recipes.

(Simon is officially on leave now, so he may want to weigh in next month)

Reply:
sure. since "recipe1" is a name of the [encoding_recipe](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/master/sre-recipes/recipes/encoding_recipe/encoding_recipe.py#L30) i will set the name to be "recipe2".